static/styles/createEventStyleSheet.css

```diff
@@ -0,0 +1,219 @@
+/*----------------------*/
+/*    general styles    */
+/*----------------------*/
+/*#region#*/
+/*
+* general user form styles
+*/
+main {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    padding: 0;
+}
+  
+body {
+  overflow-y: auto;
+}
+
+
+.create-event {
+    align-items: center;
+    -webkit-backdrop-filter: blur(5px);
+    backdrop-filter: blur(5px);
+    background: white;
+    box-sizing: border-box;
+    display: flex;
+    flex-flow: column;
+    flex: 1 2 auto;
+    font-family: 'Montserrat', sans-serif;
+    font-weight: 500;
+    height: 100%;
+    padding: 30px;
+    width: 100%;
+    min-width: 200px;
+    max-width: 350px;
+}
+
+.create-event h1 {
+    font-size: 20pt;
+}
+
+
+/*
+* form items
+*/ 
+.create-event .content-box {
+    /* height: 120px; */
+    position: relative;
+    width: 100%;
+    margin-top: 35px;
+}
+/* 
+* all inputs 
+*/
+.create-event input {
+    border-style: none;
+    box-sizing: border-box;
+    font-family: 'Montserrat', sans-serif;
+    font-size: 14pt;
+    font-weight: 500;
+    height: 30px;
+    margin: 5px 0 4px;
+    width: 100%;
+}
+
+/*
+* text/password inputs
+*/
+.create-event input[type='text'], 
+.create-event input[type='password'] {
+    border-style: none;
+    outline: 2px solid #a7a7a7;
+    padding: 0 26px 0 4px;
+}
+
+.create-event input[type='text']:focus, 
+.create-event input[type='password']:focus {
+    outline: 2px solid #cfcfcf;
+}
+
+.create-event input[type='text'].invalid-field,
+.create-event input[type='password'].invalid-field {
+    outline: 2px solid red;
+}
+
+.create-event input[type='text'].valid-field, 
+.create-event input[type='password'].valid-field{
+    outline: 2px solid #7b82b3;
+}
+
+/*
+* submit button 
+*/
+.create-event input[type='submit'] {
+    background-color: #3A3E5E;
+    border-radius: 5px;
+    color: white;
+    height: 50px;
+    -webkit-transition: 0.3s;
+    -moz-transition: 0.3s;
+    -ms-transition: 0.3s;
+    -o-transition: 0.3s;
+    transition: 0.3s;
+    width: 100%;
+}
+
+.create-event input[type='submit']:hover:not(.disable-submit) {
+    background-color: #7b82b3;
+    cursor: pointer;
+}
+
+.create-event input[type='submit'].disable-submit {
+    opacity: 0.5;
+}
+
+/*
+* labels/counters/requirement text
```

Comment:
the labels inside of the input boxes don't disappear or move when typing.

Reply:
Yeah, the labels moving is part of the frontend validation scripts - I actually do have it fixed in the frontend_validation branch, I tried to keep this pull req strictly design stuff. The labels will move in my next pull req from frontend_validation